### PR TITLE
Wrapper for bcftools split-vep plugin

### PR DESCRIPTION
**Pull request by DM2zzion**
FOR CONTRIBUTOR:
* [X] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [X] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [X] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

Wrapper for the previously unwrapped bcftools plugin split-vep

### CODE REVIEW

File: tools/bcftools/bcftools_plugin_split_vep.xml [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+@OUTPUT_TYPE@
+
+## Primary Input/Outputs
+@INPUT_FILE@
+## Plugin options
+-a '${a}'
+#if $c:
+    -c '${c}'
+#end if
```
Comment (wm75) [inline tools/bcftools/bcftools_plugin_split_vep.xml:35]: Why the `#if` when -c seems to be required?

File: tools/bcftools/bcftools_plugin_split_vep.xml [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+@EXCLUDE@
+@REGIONS@
+@TARGETS@
+
+@OUTPUT_TYPE@
+
+## Primary Input/Outputs
+@INPUT_FILE@
+## Plugin options
+-a '${a}'
```
Comment (wm75) [inline tools/bcftools/bcftools_plugin_split_vep.xml:32]:
```suggestion
-a '$a'
```
remove the curly braces here and around the other parameters. They are required only in cases where cheetah wouldn't be able to figure out on its own where a variable name ends and something else begins. In simple cases like this one they only make things less readable.

**Review by wm75 — COMMENTED** [review]
Thanks @DM2zzion, this looks really good already!
Just minor requests/questions from me.

Also, I know that you need this plugin for your work so I'm happy to get this merged even if the current PR should not offer all options that the command line tool has to offer.
In case some command line functionality is still missing (haven't checked) could you please state it here (e.g. as a TO DO in the wrapper) so that we remember to add it when we touch the wrapper next time?

File: tools/bcftools/bcftools_plugin_split_vep.xml (hunk 1 above)
Comment (DM2zzion) [reply tools/bcftools/bcftools_plugin_split_vep.xml:35]: At first I thought it's optional and when I changed it, I forgot that bit.

**Review by wm75 — APPROVED** [review]
Perfect, thanks for the changes!

**Pull request merged**